src/data/markdown/docs/30 xk6-browser/01 xk6-browser/02 BrowserContext/setOffline.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: 'setOffline(offline)'
-excerpt: "Toggles the browser's connectivity on/off."
+excerpt: "Toggles the BrowserContext's connectivity on/off."
```

Comment:
I'm on the side of saying "browser context" here, instead of "BrowserContext".

Reply:
`browser context` feels too abstract and although it is the same as `BrowserContext` it might seem like something different to a new user; whereas `BrowserContext` is the object that is being altered, and so it's easier to follow along with.

Reply:
@imiric WDYT?